app/components/namespace-quota-row/template.hbs

```diff
@@ -0,0 +1,8 @@
+<td class="pr-10">
+  <div style="height: 35px;">
```

Comment:
Single inline styles are fine every once in a while but 35px is used multiple times in this PR, move to a class.